src/js/components/bulkDownload/accounts/filters/BudgetFilter.jsx

```diff
@@ -0,0 +1,264 @@
+/**
+ * BudgetFilter.jsx
+ * Created by David Trinh 9/7/18
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import * as Icons from 'components/sharedComponents/icons/Icons';
+
+const propTypes = {
+    budgetFunctions: PropTypes.array,
+    budgetSubfunctions: PropTypes.array,
+    currentBudgetFunction: PropTypes.object,
+    currentBudgetSubfunction: PropTypes.object,
+    updateFilter: PropTypes.func,
+    valid: PropTypes.bool,
+    setBudgetSubfunctionList: PropTypes.func,
+    validAgencyId: PropTypes.bool
+};
+
+export default class BudgetFilter extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            showBudgetFunctionPicker: false,
+            showBudgetSubfunctionPicker: false
+        };
+
+        this.toggleBudgetFunctionPicker = this.toggleBudgetFunctionPicker.bind(this);
+        this.toggleBudgetSubfunctionPicker = this.toggleBudgetSubfunctionPicker.bind(this);
+        this.handleBudgetFunctionSelect = this.handleBudgetFunctionSelect.bind(this);
+        this.handleBudgetSubfunctionSelect = this.handleBudgetSubfunctionSelect.bind(this);
+    }
+
+    toggleBudgetFunctionPicker(e) {
+        e.preventDefault();
+        this.setState({
+            showBudgetFunctionPicker: !this.state.showBudgetFunctionPicker,
+            showBudgetSubfunctionPicker: false
+        });
+    }
+
+    toggleBudgetSubfunctionPicker(e) {
+        e.preventDefault();
+        this.setState({
+            showBudgetSubfunctionPicker: !this.state.showFederalAccountPicker,
```

Comment:
`!this.state.showFederalAccountPicker` should be `!this.state.showBudgetSubfunctionPicker`